tests/ui/const-generics/generic_const_exprs/const-generics-closure.rs

```diff
@@ -0,0 +1,31 @@
+// compile-fail
```

Comment:
Problem: this doesn't do anything. Compiletest directives are `//@`-prefixed (see https://rustc-dev-guide.rust-lang.org/tests/directives.html). ui tests by default are `check-fail`, you can probably just remove this.

Reply:
thanks for review! I'll remove it :)